db.py

```diff
@@ -426,3 +426,26 @@ def get_all_spawn_coords(session, pokemon_id=None):
     if config.REPORT_SINCE:
         points = points.filter(Sighting.expire_timestamp > get_since())
     return points.all()
+
+def get_known_spawnpoints(session):
+    query = ('''
+        SELECT lat, lon
+        FROM sightings
+        WHERE
+            lat >= {lat_start} AND 
+            lat <= {lat_end} AND
+            lon >= {lon_start} AND
+            lon <= {lon_end}
+        GROUP BY spawn_id;
```

Comment:
hm, I'm not really conviced he's talking about the id of spawn points. He only mentions "spawn id" once and from the context it seems like he's talking about the encounter_id field.
In our db, spawn_id is spawnpoint id, not encounter_id, right?